test/test_fx.py

```diff
@@ -1139,6 +1139,23 @@ def forward(self):
         m = M()
         self.checkGraphModule(m, ())
 
+    def test_torchbind_class_attribute_in_fx(self):
+        try:
```

Comment:
You can replace this try/catch with this incantation:

https://github.com/pytorch/pytorch/blob/fadec77c301d21e04cbc1d9652f2cf29dc4e7572/test/test_fx.py#L300